asset/cloud-client/quickstart_searchalliampolicies.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+
+
+def search_all_iam_policies(scope, query=None, page_size=None):
+    # [START asset_quickstart_search_all_iam_policies]
+    from google.cloud import asset_v1
+
+    client = asset_v1.AssetServiceClient()
+    response = client.search_all_iam_policies(
+        scope, query=query, page_size=page_size)
+    for page in response.pages:
+        for policy in page:
+            print(policy)
+        break
```

Comment:
Did you mean to leave the `break` here?

Reply:
Yes, for this sample code I just want to print the first page, as there might be too many results.